tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -27,6 +27,8 @@ LIBC=${LIBC:-musl}
 # The kata agent enables seccomp feature.
 # However, it is not enforced by default: you need to enable that in the main configuration file.
 SECCOMP=${SECCOMP:-"yes"}
+# The kata agent enables sealed-secret feature.
+SEALED_SECRET=${SEALED_SECRET:-"yes"}
```

Comment:
we may need set `SEALED_SECRET` default value to false, and enable it in `kata-deploy-binaries.sh` for `install_cc_image`